scripts/manage-db.py

```diff
@@ -84,13 +85,42 @@ def cleanup_releases_history(trans, dryrun=True):
 
     print "Total Deleted: %d" % total_deleted
 
+
+def extract_active_data(URL, loc="."):
+    """
+    Function was added in to enhance testing in Balrog's stage environment.
+
+    :param URL: Database. eg : mysql://balrogadmin:balrogadmin@balrogdb/balrog
+    :param loc: location where sqldump file must be created
+    :return: Stores sqldump data in dump.sql file in the specified location, if not specified stores in current directory
+            If dump.sql file already exists it will override that file and not append it.
+    """
+    data = re.split(':|/|@', URL)
+    user = data[3]
+    password = data[4]
+    host = data[5]
+    database = data[6]
+
+    popen('mysqldump -h %s -u %s -p%s --single-transaction %s rules rules_history rules_scheduled_changes \
+           rules_scheduled_changes_history permissions permissions_history migrate_version  > %s/dump.sql' % (host, user, password, database, loc,))
+    popen('mysqldump -h %s -u %s -p%s --single-transaction %s releases \
+           --where="exists (select * from rules where releases.name = rules.mapping)" >> %s/dump.sql ' % (host, user, password, database, loc,))
+
+    popen('mysqldump -h %s -u %s -p%s --single-transaction %s releases_history \
+    --where="releases_history.name=\'Firefox-mozilla-central-nightly-latest\' ">> %s/dump.sql' % (host, user, password, database, loc,))
+
+    popen('mysqldump -h %s -u %s -p%s --single-transaction %s releases_history --where="exists (select * from rules where \
+    releases_history.name = rules.mapping AND rules.alias LIKE \'firefox-release\' ) " >> %s/dump.sql' % (host, user, password, database, loc,))
```

Comment:
Since we know the exact string we're matching, it's better (and more performant) to use rules.alias="firefox-release" instead of LIKE here.